tests/Collection/CollectionFunctionalTest.php

```diff
@@ -311,6 +311,353 @@ public function testMapReduce()
+            [
+                function($collection, $session, $options = []) {
+                    $collection->count(
+                        [],
+                        ['session' => $session] + $options
+                    );
+                }, 'r'
+            ],
+            */
+
+            [
+                function($collection, $session, $options = []) {
+                    $collection->countDocuments(
+                        [],
+                        ['session' => $session] + $options
+                    );
+                }, 'r'
+            ],
+
+            /* Disabled, as it's illegal to use createIndex() in transactions
+            [
+                function($collection, $session, $options = []) {
+                    $collection->createIndex(
+                        ['test' => 1],
+                        ['session' => $session] + $options
+                    );
+                }, 'w'
+            ],
+            */
+
+            [
+                function($collection, $session, $options = []) {
+                    $collection->deleteMany(
+                        ['test' => 'foo'],
+                        ['session' => $session] + $options
+                    );
+                }, 'w'
+            ],
+
+            [
+                function($collection, $session, $options = []) {
+                    $collection->deleteOne(
+                        ['test' => 'foo'],
+                        ['session' => $session] + $options
+                    );
+                }, 'w'
+            ],
+
+            [
+                function($collection, $session, $options = []) {
+                    $collection->distinct(
+                        '_id',
+                        [],
+                        ['session' => $session] + $options
+                    );
+                }, 'r'
+            ],
+
+            /* Disabled, as it's illegal to use drop() in transactions
+            [
+                function($collection, $session, $options = []) {
+                    $collection->drop(
+                        ['session' => $session] + $options
+                    );
+                }, 'w'
+            ],
+            */
+
+            /* Disabled, as it's illegal to use dropIndexes() in transactions
+            [
+                function($collection, $session, $options = []) {
+                    $collection->dropIndex(
+                        '_id_1',
+                        ['session' => $session] + $options
+                    );
+                }, 'w'
+            ], */
+
+            /* Disabled, as it's illegal to use dropIndexes() in transactions
+            [
+                function($collection, $session, $options = []) {
+                    $collection->dropIndexes(
+                        ['session' => $session] + $options
+                    );
+                }, 'w'
+            ],
+            */
+
+            /* Disabled, as count() can't be used in transactions
```

Comment:
I assume this will ultimately be deleted, but if not the comment should probably refer to `count` command for `estimatedDocumentCount()` method.